setup.py

```diff
@@ -57,7 +57,7 @@
     'long_description_content_type': "text/markdown",
     'provides': ["pennylane"],
     'install_requires': requirements,
-    'extras_require': {'kernels': ['cvxpy==1.1.11']},
+    'extras_require': {'kernels': ['cvxpy==1.1.11', 'cvxopt==1.2.6']},
```

Comment:
Why not `>=`? Is there a reason this specific version is required?

Reply:
Ideally, the `setup.py` file should have no pinning, or minimal pinning if required. This is to avoid placing too many restrictions on a users environment. So simply having `['cvxpy', 'cvxopt']` should be sufficient!

Reply:
I will take care of this, then!